node-backend/app/cronJobs/ResetHi5QuotaBalance.js

```diff
@@ -1,6 +1,60 @@
+const log4js = require("log4js");
 var cron = require("node-cron");
+const moment = require("moment");
 
 const db = require("../models/sequelize");
+const Organizations = db.organizations;
+const Users = db.users;
+
+const now = moment();
+const logger = log4js.getLogger();
+
 cron.schedule(process.env.TIME_RESET_HI5_QUOTA, async () => {
-  db.sequelize.query("select resetHi5Quota();");
+  try {
+    const noOfOrganisation = await Organizations.count();
+    let iterator = 1;
+    while (iterator < noOfOrganisation + 1) {
+      try {
+        const organisationObject = await Organizations.findOne({
+          where: { id: iterator },
```

Comment:
organisation ids will not be in sequence always. so iterator with 1, 2, 3 .. is not valid